hello-world/clients.js

```diff
@@ -0,0 +1,75 @@
+var express = require('express');
+var Client = require('node-rest-client').Client;
+var client = new Client();
+var router = express.Router();
+var rp = require('request-promise');
+var request = require('request');
+module.exports = router;
+
+router.get('/', function(req, res, next) {
+  request.get('https://api.createsend.com/api/v3.1/clients.json', {
+    'auth': {
```

Comment:
@psinha20 Use separate config file for storing the details

```
'auth': {
       'user': '9a86a35217185b745506196eb27709a0',
       'pass': 'x'
 }
```

so create the file config.js and write the above code into that file and export the object and also use ''https://api.createsend.com/api/v3.1/clients/' in the config file.